packages/index.ts

```diff
@@ -95,6 +95,12 @@ export {
   NumberCell
 } from "./table";
 export { TableView, TableViewHeader, TableViewBody } from "./tableView";
+export {
+  Table as TableV2,
```

Comment:
I wonder if we should rename Table as TableV1 or Table_Deprecated and then do a major version release so we don't have to live with TableV2 as a name forever. Let clients rename if they want to keep the old table.

Reply:
i like that approach.

Reply:
I'm on board with this 👍